python/cudf/cudf/utils/_numba.py

```diff
@@ -65,7 +78,7 @@ def _get_ptx_file(path, prefix):
         return regular_result[1]
 
 
-def _patch_numba_mvc():
+def patch_numba_linker_cuda_11():
```

Comment:
Maybe this should be named by the package being used rather than the major version? But I'm okay with this one saying CUDA 11 since it really is specific to that major version.
```suggestion
def patch_numba_linker_cubinlinker():
```